app/cmake/version.cmake

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+unset(SCITT_VERSION)
+
+if (SCITT_VERSION_OVERRIDE)
+  set(SCITT_VERSION "${SCITT_VERSION_OVERRIDE}")
+else()
+  # Deduce project version from git environment
+  find_package(Git)
+
+  execute_process(
+    COMMAND "bash" "-c" "${GIT_EXECUTABLE} describe --tags --match=\"*.*.*\""
```

Comment:
Is `bash -c` necessary? Wouldn't just `COMMAND "${GIT_EXECUTABLE}" "describe" "--tags" "--match=*.*.*"` work?